static/templates/recipient_row.hbs

```diff
@@ -42,7 +42,7 @@
                     <i class="fa fa-pencil always_visible_topic_edit recipient_bar_icon" {{#unless realm_allow_message_editing}}style="display: none"{{/unless}} role="button" tabindex="0" aria-label="{{t 'Edit' }}"></i>
                 {{else}}
                     {{#if on_hover_topic_edit}}
-                    <i class="fa fa-pencil on_hover_topic_edit recipient_bar_icon" {{#unless realm_allow_message_editing}}style="display: none"{{/unless}} role="button" tabindex="0" aria-label="{{t 'Edit' }}"></i>
+                    <i class="fa fa-pencil on_hover_topic_edit recipient_bar_icon" {{#unless realm_allow_message_editing}}style="display: none"{{/unless}} role="button" title="{{t 'Edit'}}" tabindex="0" aria-label="{{t 'Edit' }}"></i>
```

Comment:
Can you put the `title` attribute before the `role` attribute? Just to maintain consistency of the order of attributes across elements.

Also, add a single space before ending curly braces.

Reply:
Done! @garg3133